Library/Homebrew/test/dev-cmd/livecheck_spec.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require "cmd/shared_examples/args_parse"
+
+describe "Homebrew.livecheck_args" do
+  it_behaves_like "parseable arguments"
+end
+
+describe "brew livecheck", :integration_test do
+  it "errors when no Formula arguments are given and watchlist is empty" do
+    expect { brew "livecheck" }
+      .to output(/Invalid usage: No formulae to check./).to_stderr
```

Comment:
Thoughts on having this check a single, specified formula?

Reply:
I was thinking of checking the `ruby` Formula, but since upstream is unreliable, crazy idea (I don't know how feasible it is though): how about a 'fake' temporary Formula that checks for Homebrew/brew's latest version or something?